src/mono/nuget/Microsoft.NET.Workload.Mono.Toolchain.Current.Manifest/WorkloadTelemetry.targets

```diff
@@ -0,0 +1,45 @@
+<!-- Licensed to the .NET Foundation under one or more agreements. The .NET Foundation licenses this file to you under the MIT license. -->
+<Project>
+  <Target Name="_WorkloadTelemetry"
+          Condition="'$(_MonoWorkloadTargetsMobile)' == 'true' and '$(TargetsNet8)' == 'true'"
```

Comment:
Is the `TargetsNet8` instead of `TargetsCurrent` deliberate here?

Reply:
Yes, I was thinking it would help when we switch to .NET 9, but then again, maybe it's easier to flip all `TargetsCurrent` properties at that point.